rollup-interface/src/state_machine/mocks/da.rs

```diff
@@ -12,8 +10,12 @@ use serde::{Deserialize, Serialize};
 use crate::da::{
     BlobReaderTrait, BlockHashTrait, BlockHeaderTrait, CountedBufReader, DaSpec, DaVerifier, Time,
 };
+use crate::maybestd::string::String;
+#[cfg(all(feature = "native", feature = "tokio"))]
+use crate::maybestd::sync::Arc;
```

Comment:
Similarly, `alloc::sync::Arc` would probably work just fine here.

Reply:
The problem of requiring that is we introduce the need to `extern crate alloc` on every library. They all compile to the same code, it's just a matter of what is more convenient. Wdyt?

Reply:
I personally think `extern crate alloc` is ideal, it simplifies the API surface area at the cost of just 1 extra line of code.